### PR TITLE
fix(deployments): Fix sub-service name for HA echo in warning m…

### PR DESCRIPTION
**Pull request by dragon3**
This pull request just fixes the sub-service name for HA echo in the warning message.

### CODE REVIEW

File: halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/HaEchoValidator.java [hunk 1]
```diff
@@ -38,7 +38,7 @@ public void validate(ConfigProblemSetBuilder p, DeploymentEnvironment n) {
       p.addProblem(
           Problem.Severity.WARNING,
           "High Availability (HA) is enabled for echo, but found custom sizing for the main service (this setting will be ignored). "
-              + "With HA enabled, the service is split into multiple sub-services (echo-scheduler, echo-scheduler). You need to update the component sizing for each sub-service, individually.");
+              + "With HA enabled, the service is split into multiple sub-services (echo-scheduler, echo-worker). You need to update the component sizing for each sub-service, individually.");
```
Comment (dragon3) [inline halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/HaEchoValidator.java]: [note]
You can see the sub-service names:
https://github.com/spinnaker/halyard/blob/d8af98d3dddf3dd420f64534895a1dd7ee85efae/halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/HaEchoValidator.java#L45-L46

Comment (german-muzquiz) [reply halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/HaEchoValidator.java]: How about specifying the full service name in the warning message as it is configured in customSizing: `spin-echo-scheduler` and `spin-echo-worker`?

Comment (dragon3) [reply halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/HaEchoValidator.java]: Oh, sure! 
I'll do it 👍

Comment (dragon3) [reply halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/HaEchoValidator.java]: I've done 4fdeae2

**Comment by dragon3** [comment]
@german-muzquiz 
Thank you for your review and comment.

I think this pull request is ready to merge, but let me know if I need to anything more.

Thanks!

**Comment by german-muzquiz** [comment]
cc @ethanfrogers

**Comment by dragon3** [comment]
Thanks!

**Pull request merged**